multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -514,13 +514,10 @@ def VOCName2Idx(name):
     return VOC_NAME_TO_IDX[name]
 
 
-def get_image_name_num(path):
-    if "/potholes" in path:
-        start_idx = path.rfind("/potholes") + 9
-    else:
-        start_idx = path.rfind("/") + 1
+def get_image_filename(path):
```

Comment:
One comment is, for functions inside utils.py, it is also better to include a docstring (and add an example would be the best). Because these functions might be reused in a lot of scenarios, users need to know what the function is doing.

Reply:
Done for function modified here. I'll use another PR to go through all the object utils.